examples/stable_diffusion_xl/gm/helpers.py

```diff
@@ -217,8 +227,8 @@ def create_model(
     return model, None
 
 
-def get_grad_reducer(is_parallel, parameters):
-    if is_parallel:
+def get_grad_reducer(args, is_parallel, parameters):
+    if is_parallel and args.parallel_mode == "DATA_PARALLEL":
```

Comment:
Don't pass all arguments just to extract one variable. Pass only the `parallel_mode` argument.

Reply:
fixed